server/handlers/tokenHandler/schemas.js

```diff
@@ -1,15 +1,14 @@
 const Joi = require('joi');
 
 const tokenGetSchema = Joi.object({
-  limit: Joi.number().min(1).max(1000).required().default(1000),
-  offset: Joi.number().min(0).integer().default(0),
+  limit: Joi.number().integer().min(1).max(2000).default(2000),
+  offset: Joi.number().integer().min(0).default(0),
   wallet: Joi.string(),
 });
 
 const tokenGetTransactionsByIdSchema = Joi.object({
-  limit: Joi.number().min(1).max(1000).integer().default(1000).required(),
-  offset: Joi.number().min(0).integer(),
-  id: Joi.string().guid(),
```

Comment:
removed the id?

Reply:
Yes. The ID of the token is read from `req.params`. The `id` here in `req.query` is not needed. It is never used.